src/main/java/Victor/TaskList/TaskList.java

```diff
@@ -0,0 +1,49 @@
+package Victor.TaskList;
+
+import Victor.TaskType.Task;
+import Victor.Ui.Ui;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    ArrayList<Task> currentTaskList;
+    Ui ui = new Ui();
+    public TaskList(ArrayList<Task> currentTaskList) {
+        this.currentTaskList = currentTaskList;
+    }
+
+    public TaskList() {
+        this.currentTaskList = new ArrayList<Task>();
+    }
+
+    public void printList() {
```

Comment:
I like how all method names for this pr are all verbs.

Reply:
Thanks